templates/node/package.json.twig

```diff
@@ -14,7 +14,7 @@
     "@types/node": "^18.16.1"
   },
   "dependencies": {
-    "axios": "^1.4.0",
+    "axios": "^1.6.3",
```

Comment:
Is this supposed to be part of this PR? Btw, i think we are planning on getting rid of axios.

Reply:
reverted for now